src/test/objectstore/allocator_replay_test.cc

```diff
@@ -18,8 +18,14 @@
 using namespace std;
 
 void usage(const string &name) {
-  cerr << "Usage: " << name << " <log_to_replay> <raw_duplicate|free_dump|try_alloc count want alloc_unit>"
-       << std::endl;
+  cerr << "Usage: " << name << " <log_to_replay> <raw_duplicate|free_dump|try_alloc count want alloc_unit|replay_alloc alloc_list_file>" << std::endl;
+}
+
+void usage_replay_alloc(const string &name) {
+  cerr << "Detailed replay_alloc usage: " << name << " <allocator_dump_JSON> <replay_alloc> <alloc_list_file> [number of replays]" << std::endl;
```

Comment:
IMO better show <replay_alloc> which is a static keyword differently from other parameter names, e.g
... <allocator_dump_JSON> replay_alloc <alloc_list_file>....

Reply:
Thanks, I missed this.